x/src/config.rs

```diff
@@ -0,0 +1,47 @@
+use crate::{utils::project_root, Result};
+use serde::{Deserialize, Serialize};
+use std::{
+    collections::HashMap,
+    fs,
+    path::{Path, PathBuf},
+};
+
+#[derive(Clone, Serialize, Deserialize, Debug, PartialEq, Eq)]
+#[serde(rename_all = "kebab-case")]
+pub struct Config {
+    /// Package exceptions which need to be run special
+    package_exceptions: HashMap<String, Package>,
+}
+
+#[derive(Clone, Serialize, Deserialize, Debug, PartialEq, Eq)]
+#[serde(rename_all = "kebab-case")]
+pub struct Package {
+    path: PathBuf,
+    #[serde(default)]
+    pub all_features: bool,
```

Comment:
It seems s lightly odd that default for this would be false. I think you want `#[serde(default = true)]` for this one.

Reply:
Oh good point, let me fix that

Reply:
Turns out this isn't as easy as saying "default = true" yet: https://github.com/serde-rs/serde/issues/368 but there's an easy workaround.